tests/test_reader_scenarios.py

```
@@ -143,13 +149,17 @@ def test_reader_search_to_object(temp_vault):
         st, search = _get(port, "/search?q=Alpha")
         assert st == 200
         assert "Search" in search
-        assert 'href="/ops' not in search
+        # Reader page may carry a single cross-link to /ops, but
+        # never a deep link into the Maintainer subtree.
+        assert 'href="/ops/' not in search
 
         result_links = _extract_hrefs(search, "/object?")
         assert result_links, "search results should contain /object? links"
         st, obj = _get(port, result_links[0])
```

Ruff (0.15.12)

[warning] Unpacked variable `obj` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)